projects/configs/tag-operations/test-extract-uclh-omop-cdm.yaml

```diff
@@ -199,6 +223,18 @@
   group: 0x0010
   element: 0x2110
   op: "delete"
+- name: "Additional Patient History"
+  group: 0x0010
+  element: 0x21b0
+  op: "delete"
+- name: "Pregnancy Status"
+  group: 0x0010
+  element: 0x21c0
+  op: "delete"
+- name: "Ethnic Group"
+  group: 0x0010
+  element: 0x2160
+  op: "delete"
```

Comment:
We can just delete these from the yaml as we apply an allowlist approach. Same for the other deletes

Reply:
Thanks, I have started here [1f8a309](https://github.com/SAFEHR-data/PIXL/pull/493/commits/1f8a3095baea9bc09cb5b07e6c3d37b2b043f82e). Do we need to remove `delete` operation tags from all config files?

Reply:
Probably, only time we want to keep delete is if we're over-writing a default. so for instance a procject could have `base.yml`, `mri.yml` and then `project.yml` defined, and it might have a delete in the `project.yml`

Reply:
Thanks, just removed all tags with `delete` operations [2d492ae](https://github.com/SAFEHR-data/PIXL/pull/493/commits/2d492aee2f930583af2b4910dfbb111d12bf40dc)